### PR TITLE
add online-hibernation test job

### PR DESCRIPTION
**Pull request by sallyom**
@stevekuznetsov i've changed to inherit from origin.yml and removed hack/env.  thanks for your review.

### CODE REVIEW

File: sjb/config/common/test_cases/online_hibernation.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (stevekuznetsov) [inline sjb/config/common/test_cases/online_hibernation.yml]: I would inherit from the Origin one, you get integration with Gubernator there and parameters, etc

File: sjb/config/common/test_cases/online_hibernation.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+parameters: []
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+sync_repos:
+    - name: "online-hibernation"
```
Comment (stevekuznetsov) [inline sjb/config/common/test_cases/online_hibernation.yml]: syntax error?

File: sjb/config/test_cases/test_branch_online_hibernation_integration.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+overrides:
+  sync_repos:
+    - name: "online-hibernation"
+extensions:
+  actions:
+    - type: "script"
+      title: "run online-hibernation integration tests"
+      repository: "online-hibernation"
+      script: |-
+        OS_BUILD_ENV_TMP_VOLUME='/tmp' hack/env JUNIT_REPORT='true' make test-integration
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_online_hibernation_integration.yml]: You guys don't have `hack/env`, right?

File: sjb/config/common/test_cases/online_hibernation.yml (hunk 1 above)
Comment (sallyom) [reply sjb/config/common/test_cases/online_hibernation.yml]: ok, thanks

**Comment by sallyom** [comment]
@stevekuznetsov we have only unit-tests thus far for online-hibernation.  Is this all that's necessary to get up and running with ci? 
thanks

File: sjb/config/test_cases/test_branch_online_hibernation_unit.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_online_hibernation_unit.yml]: You need to sync the repo in this job, but otherwise this seems reasonable

Comment (sallyom) [reply sjb/config/test_cases/test_branch_online_hibernation_unit.yml]: done, and added a slew of generated files, look good?

**Comment by sallyom** [comment]
@stevekuznetsov if this looks good, I'll follow suit with openshift/online-analytics|online-registration

File: sjb/config/common/test_cases/origin.yml [hunk 5]
```diff
@@ -25,6 +25,7 @@ provision:
 sync_repos:
   - name: "origin"
   - name: "image-registry"
+  - name: "online-hibernation"
```
Comment (stevekuznetsov) [inline sjb/config/common/test_cases/origin.yml]: Err -- sorry -- you should add this to the overrides list in your jobs, but we should not ask all Origin jobs to clone down this repo!

File: sjb/config/test_cases/test_branch_online_hibernation_unit.yml (hunk 4 above)
Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_online_hibernation_unit.yml]: So here you'd see:

```yaml
overrides:
  sync_repos:
    - name: "online-hibernation"
```

Comment (sallyom) [reply sjb/config/test_cases/test_branch_online_hibernation_unit.yml]: Got it, lol, it didn't seem right to add to origin.yml, but that's what I interpreted from your comment. *sigh* 
Do I have it right now? 
Thanks

**Comment by stevekuznetsov** [comment]
Live:

https://ci.openshift.redhat.com/jenkins/job/test_pull_request_online_hibernation_unit/
https://ci.openshift.redhat.com/jenkins/job/test_branch_online_hibernation_unit/

**Pull request merged**